keras_cv/models/mobilenet_v3.py

```diff
@@ -0,0 +1,534 @@
+
+    Args:
+        divisor: integer, the value by which a channel number should be divisble,
+            defaults to 8.
+        min_value: float, minimum value for the new tensor.
+        name: string, layer label.
+
+    Returns:
+        a function that takes an input Tensor representing a Depth layer.
+    """
+    if name is None:
+        name = f"depth_{backend.get_uid('depth')}"
+
+    if min_value is None:
+        min_value = divisor
+
+    def apply(x):
+        new_x = max(min_value, int(x + divisor / 2) // divisor * divisor)
+
+        # make sure that round down does not go down by more than 10%.
+        if new_x < 0.9 * x:
+            new_x += divisor
+        return new_x
+
+    return apply
+
+
+def HardSigmoid(name=None):
+    """The Hard Sigmoid function.
+
+    Args:
+        name: string, layer label.
+
+    Returns:
+        a function that takes an input Tensor representing a HardSigmoid layer.
+    """
+    if name is None:
+        name = f"hard_sigmoid_{backend.get_uid('hard_sigmoid')}"
+
+    activation = layers.ReLU(6.0)
+
+    def apply(x):
+        return activation(x + 3.0) * (1.0 / 6.0)
+
+    return apply
+
+
+def HardSwish(name=None):
+    """The Hard Swish function.
+
+    Args:
+        name: string, layer label.
+
+    Returns:
+        a function that takes an input Tensor representing a HardSwish layer.
+    """
+    if name is None:
+        name = f"hard_swish_{backend.get_uid('hard_swish')}"
+
+    hard_sigmoid = HardSigmoid()
+    multiply_layer = layers.Multiply()
+
+    def apply(x):
+        return multiply_layer([x, hard_sigmoid(x)])
+
+    return apply
+
+
+def InvertedResBlock(
+    expansion, filters, kernel_size, stride, se_ratio, activation, block_id, name=None
+):
+    """An Inverted Residual Block.
+
+    Args:
+        expansion: integer, the expansion ratio, multiplied with infilters to get the
+            minimum value passed to Depth.
+        filters: integer, number of filters for convolution layer.
+        kernel_size: integer, the kernel size for DpethWise Convolutions.
+        stride: integer, the stride length for DpethWise Convolutions.
+        se_ratio: float, ratio for bottleneck filters. Number of bottleneck
+            filters = filters * se_ratio.
+        activation: the activation layer to use.
+        block_id: integer, a unique identification if you want to use expanded
+            convolutions.
+        name: string, layer label.
+
+    Returns:
+        a function that takes an input Tensor representing a InvertedResBlock.
+    """
+    if name is None:
+        name = f"inverted_res_block_{backend.get_uid('inverted_res_block')}"
+
+    def apply(x):
+        shortcut = x
+        prefix = "expanded_conv/"
+        infilters = backend.int_shape(x)[channel_axis]
+
+        if block_id:
+            prefix = "expanded_conv_{}/".format(block_id)
+
+            x = layers.Conv2D(
+                Depth()(infilters * expansion),
```

Comment:
once you update Depth, change Depth()(...) => depth(...).

Reply:
Done, this everywhere we use `Depth()`